CMP1903_A1_2324/Die.cs

```diff
@@ -8,15 +8,22 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
+        
 
         //Method
+        public int holdValue { get; private set; } //to store the value gotten from rolling the dice
+        // <black-eye-inc> Code Review:
```

Comment:
I'd recommend using _camelCase (calling it "_holdValue") for private parameters (as instructed by "C# style guidelines")
One of the Object-Oriented Principles is encapsulation (making all parameters (within reason) private).